shell/platform/fuchsia/flutter/accessibility_bridge.cc

```diff
@@ -309,15 +320,62 @@ void AccessibilityBridge::AddSemanticsNodeUpdate(
     PrintNodeSizeError(nodes.back().node_id());
   }
 
+  // Even if the SemanticsNodeUpdates does not contain the root node, we still
+  // need to generate an update for the root node in case the view pixel ratio
+  // has changed.
+  std::vector<fuchsia::accessibility::semantics::Node> root_update;
+  root_update.push_back(GetRootNodeUpdate());
```

Comment:
We should avoid sending an update to the root node if it has not changed (i.e. if the DPR has not changed). Sending the FIDL message is almost certainly more expensive than trackinging the state.

It's also not clear to me why we'd send a separate update instead of just including the root node in the update.